newIDE/app/src/AssetStore/AssetStoreContext.js

```diff
@@ -77,6 +115,73 @@ export const AssetStoreStateProvider = ({
   const [searchText, setSearchText] = React.useState(defaultSearchText);
   const filtersState = useFilters();
 
+  const [
+    animatedFilter,
+    setAnimatedFilter,
+  ] = React.useState<AnimatedAssetStoreSearchFilter>(
+    new AnimatedAssetStoreSearchFilter()
+  );
+  const [
+    viewpointFilter,
+    setViewpointFilter,
+  ] = React.useState<TagAssetStoreSearchFilter>(
+    new TagAssetStoreSearchFilter()
+  );
+  const [
+    dimensionFilter,
+    setDimensionFilter,
+  ] = React.useState<DimensionAssetStoreSearchFilter>(
+    new DimensionAssetStoreSearchFilter()
+  );
+  const [
+    objectTypeFilter,
+    setObjectTypeFilter,
+  ] = React.useState<ObjectTypeAssetStoreSearchFilter>(
+    new ObjectTypeAssetStoreSearchFilter()
+  );
+  const [
+    colorFilter,
+    setColorFilter,
+  ] = React.useState<ColorAssetStoreSearchFilter>(
+    new ColorAssetStoreSearchFilter()
+  );
+  const [
+    licenseFilter,
+    setLicenseFilter,
+  ] = React.useState<LicenseAssetStoreSearchFilter>(
+    new LicenseAssetStoreSearchFilter()
+  );
+  const assetFiltersState = {
```

Comment:
This object will be a new one at each render, do you need to `useMemo` on it? 
Otherwise it's passed to `const assetStoreState = React.useMemo(` and will make it change at each render even if the filters did not change?